### ISSUE TITLE
Accessibility -> A&R - Cannot access the Date picker using the keyboard on the NEter data page

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#263]
## Description
**How to replicate:**
* Navigate to the A&R page on the Test environment and log in : [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/enter-data?date=Invalid%20date)
* Navigate to the Enter Data page and try to fill the filters Date, Park, and Sub-areas only using the keyboard

**Expected Behavior:**
* The user should be able to navigate only using the keyboard

**Issues:**
* The date picker on the Enter data page for the “date” field is not accessible using the keyboard

  ![Image](https://github.com/bcgov/bcparks-ar-api/assets/145403218/7543bfb4-9438-4a90-86e3-6c63fc7952b0)

## Linked issues
relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
